src/Admin/SiteHealth.php

```diff
@@ -335,29 +335,31 @@ static function ( $available_service ) {
 	 * @return array
 	 */
 	public function page_cache() {
-		$has_page_caching = $this->has_page_caching();
+		$page_cache_status = $this->get_page_cache_status();
 
-		$badge_color = 'orange';
-		$status      = 'recommended';
+		$badge_color = 'red';
+		$status      = is_wp_error( $page_cache_status ) ? 'critical' : $page_cache_status;
 		$label       = __( 'Page caching is not detected', 'amp' );
 
 		$description = '<p>' . esc_html__( 'The AMP plugin performs at its best when page caching is enabled. This is because the additional optimizations performed require additional server processing time, and page caching ensures that responses are served quickly.', 'amp' ) . '</p>';
 
 		/* translators: 1 is Cache-Control, 2 is Expires, and 3 is Age */
 		$description .= '<p>' . sprintf( __( 'Page caching is detected by making three requests to the homepage and looking for %1$s, %2$s, or %3$s HTTP response headers.', 'amp' ), '<code>Cache-Control: max-age=…</code>', '<code>Expires</code>', '<code>Age</code>' );
```

Comment:
This description is no longer accurate. Perhaps something like this:

```suggestion
		/* translators: 1 is Cache-Control, 2 is Expires, and 3 is Age */
		$description .= '<p>' . sprintf( __( 'Page caching is detected by looking for an active page caching plugin, making three requests to the homepage and looking for HTTP response headers like: %s.', 'amp' ), '<code>' . implode( '</code>, <code>', array_keys( self::get_page_cache_headers() ) ) . '</code>' );
```